nibabel/gifti/tests/test_gifti.py

```diff
@@ -34,10 +34,11 @@ def test_gifti_image():
     gi = GiftiImage()
     assert_equal(gi.numDA, 0)
 
-    da = GiftiDataArray(data='data')
```

Comment:
Yes. This could be tested separately, without saving, to validate that properties are being set.